R/intervals_support_funs.R

```diff
@@ -0,0 +1,244 @@
+# Load necessary library
+library(dplyr)
```

Comment:
Please remove `library()` from this file. If we need `dplyr` functions, please use `dplyr::function_name()`.